episodes/01-basics.md

```diff
@@ -17,20 +17,60 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-We'll start by exploring how version control can be used
-to keep track of what one person did and when.
+## Tracking changes
+
+We'll start by exploring how we are usually introduced to version control to keep track of what one person did and when.
 Even if you aren't collaborating with other people,
-automated version control is much better than this situation:
+version control may have look like this situation:
+
+![Tracking changes for the `manuscript.docx` file.](fig/git-manual-version-system.png){alt='One same files called manuscript with modification dates as prefix and what a person did as sufix.'}
+
+We all started by replicating the Version control system of our initial community. We all need to differentiate files from previous versions or different colleagues (supervisors or reviewers). For those reasons, we tag each version to track what one person did (e.g., with a suffix) or when (e.g., with a prefix).
+
+Does it seem unnecessary to you to have multiple nearly identical versions of the same document? Possibly yes. But this Version control system opens the possibility of returning to a specific version in case you erased something that you think now is essential.
+
+::::::::::::::::: discussion
+
+### File names to track changes
+
+Write down:
+
+- Is there any file naming convention that is familiar to you?
 
-!["notFinal.doc" by Jorge Cham, <https://www.phdcomics.com>](fig/phd101212s.png){alt='Comic: a PhD student sends "FINAL.doc" to their supervisor, but after several increasingly intense and frustrating rounds of comments and revisions they end up with a file named "FINAL_rev.22.comments49.corrections.10.#@$%WHYDIDCOMETOGRADSCHOOL????.doc"'}
+- What was the version control system that you first used?
 
-We've all been in this situation before: it seems unnecessary to have
-multiple nearly-identical versions of the same document. Some word
+- Share with us your favorite prefix or suffix!
+
+:::::::::::::::::::::::::::::
+
+::::::::::::::::::: instructor
+
+We can use the comments or live participation as ice-breaker.
+
+::::::::::::::::::::::::::::::
+
+Some word
 processors let us deal with this a little better, such as Microsoft
 Word's
 [Track Changes](https://support.office.com/en-us/article/Track-changes-in-Word-197ba630-0f5f-4a8e-9a77-3712475e806a),
 Google Docs' [version history](https://support.google.com/docs/answer/190843?hl=en), or
-LibreOffice's [Recording and Displaying Changes](https://help.libreoffice.org/Common/Recording_and_Displaying_Changes).
+LibreOffice's [Recording and Displaying Changes](https://help.libreoffice.org/Common/Recording_and_Displaying_Changes). Let's illustrate how Google Docs works.
+
+![Tracking changes for the `git-test` file.](fig/gdocs-01.png)
```

Comment:
This screenshot is of google docs but in the image it says "Word's version control is similar to Git"

Reply:
added as an issue in #21